src/roiextractors/testing.py

```diff
@@ -11,6 +11,29 @@
 inttype = (int, np.integer)
 
 
+import numpy as np
+
+from roiextractors import NumpyImagingExtractor
+
+
+def generate_dummy_imaging_extractor(
+    num_frames: int = 100,
+    rows: int = 10,
+    columns: int = 10,
+    num_channels: int = 1,
+    sampling_frequency: float = 30000,
```

Comment:
The average imaging rate of data we've seen is around 30Hz, not 30 kHz.

Reply:
Yup!